mod_reforged/config/config.nut

```diff
@@ -1,6 +1,4 @@
 ::Reforged.Config <- {
-	HiringCostVariance = 0.2,	// Hiring cost for recruits after gear is randomized +- this value. 0.0 = no variance. 0.2 = it varies between 80% and 120% of its original cost
```

Comment:
The changes about the hiring cost variance have snuck into this PR and should be removed.